### PR TITLE
Add python39 runtime

### PR DESCRIPTION
**Pull request by moelsayed**
**Description**
Add python39 runtime
Changes proposed in this pull request:

- Add python39 runtime

**Related issue(s)**

kyma-project/kyma#11299

### CODE REVIEW

**Comment by moelsayed** [comment]
/hold
depends on https://github.com/kyma-project/kyma/pull/11498

File: core-ui/src/components/Lambdas/constants.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ export const LAMBDA_PHASES = {
 export const PRETTY_RUNTIME_NODEJS12_NAME = 'Node.js 12';
 export const PRETTY_RUNTIME_NODEJS14_NAME = 'Node.js 14';
 export const PRETTY_RUNTIME_PYTHON38_NAME = 'Python 3.8';
```
Comment (kwiatekus) [inline core-ui/src/components/Lambdas/constants.js:65]:
```suggestion
export const PRETTY_RUNTIME_PYTHON38_NAME = 'Python 3.8 - Deprecated';
```
See [here](https://github.com/kyma-project/website/pull/744/files#diff-d73d6705bfab87e27c16586adf09ece8d9674bd57bbed011c0d1266d1a509fd5R37)

**Comment by moelsayed** [comment]
Closing this for now, since there are no longer releases for 1.x.

**Pull request closed without merging**